org.freeciv.sdl2.yml

```diff
@@ -0,0 +1,48 @@
+app-id: org.freeciv.sdl2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: freeciv-sdl2
+rename-icon: freeciv-client
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.freeciv:create
+modules:
+  - name: sdl2-gfx
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/sdl2gfx/files/SDL2_gfx-1.0.4.tar.gz
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+      - type: shell
+        commands:
+          - mv configure.in configure.ac
+          - aclocal
+          - libtoolize -f -c
+          - autoconf
+          - automake -a -f -c
```

Comment:
Yes, that was the suggestion, but I thought that it would be cleaner solution to let automake update consistent (automake should now better what files belong to its own version) set of files instead of manually copying just some files. I asked about this, but you didn't reply, so I went with my own idea.

Reply:
if you are not using autoreconf then it's not clean.

this is the pattern used all accross and this is even the recommended minimal fix.

Reply:
Done